app/client/cli/root.go

```diff
@@ -84,7 +84,7 @@ func versionCheckInterceptor(ctx context.Context, method string, req interface{}
 
 	ok, err := isClientVersionLessThan(daemonVersion)
 	if err != nil {
-		return fmt.Errorf("Unable to read daemon's version: %s", err)
+		return nil
```

Comment:
Why ignore the error?

Reply:
I think, it is left when I was trying to setup local Haaukins platform, I just tried to see what happen when I ignore it. I should rewrite it down.